source/docs/advanced/features/overview.md

```diff
@@ -13,5 +13,6 @@ Front-Commerce has several extension points that allow to override default imple
 - [SmartForms](./smart-forms.html)
 - [Quick order](./quickorder.html)
 - [Password fields](./password-fields.html)
+- [In-stock alert](./in-stock-alert.html)
```

Comment:
Is it on purpose to not add it to the left menu ? 
![image](https://user-images.githubusercontent.com/91904856/166438458-683a3bea-f022-451e-aa63-fcce6dc513dd.png)

Reply:
Nope, forgot about it :+1: